src/main/java/com/spotinst/sdkjava/model/repo/azure/statefulNode/SpotinstAzureStatefulNodeRepo.java

```diff
@@ -0,0 +1,168 @@
+import com.spotinst.sdkjava.model.bl.azure.statefulNode.StatefulNodeGetStatusConfig;
+import com.spotinst.sdkjava.model.converters.azure.statefulNode.StatefulNodeConverter;
+import com.spotinst.sdkjava.model.requests.azure.statefulNode.ApiStatefulNodeStateRequest;
+import com.spotinst.sdkjava.model.requests.azure.statefulNode.StatefulNodeStateRequest;
+import com.spotinst.sdkjava.model.service.azure.statefulNode.SpotinstAzureStatefulNodeService;
+import com.sun.org.apache.xpath.internal.operations.Bool;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class SpotinstAzureStatefulNodeRepo implements ISpotAzureStatefulNodeRepo {
+
+    @Override
+    public RepoGenericResponse<StatefulNode> createNode(StatefulNode NodeToCreate, String authToken, String account) {
+
+        RepoGenericResponse<StatefulNode> retVal;
+
+        try {
+            ApiStatefulNode apiStatefulNodeToCreate = StatefulNodeConverter.toDal(NodeToCreate);
+            ApiStatefulNode apiCreatedNode          = SpotinstAzureStatefulNodeService
+                    .createNode(apiStatefulNodeToCreate, authToken, account);
+
+            StatefulNode createdStatefulNode = StatefulNodeConverter.toBl(apiCreatedNode);
+            retVal = new RepoGenericResponse<>(createdStatefulNode);
+        }
+        catch (SpotinstHttpException ex) {
+            retVal = ExceptionHelper.handleHttpException(ex);
+        }
+
+        return retVal;
+    }
+
+    @Override
+    public RepoGenericResponse<StatefulNode> getNode(String nodeId, String authToken, String account) {
+
+        RepoGenericResponse<StatefulNode> retVal;
+
+        try {
+            ApiStatefulNode apiGetNode          = SpotinstAzureStatefulNodeService
+                    .getNode(nodeId, authToken, account);
+
+            StatefulNode getStatefulNode = StatefulNodeConverter.toBl(apiGetNode);
+            retVal = new RepoGenericResponse<>(getStatefulNode);
+        }
+        catch (SpotinstHttpException ex) {
+            retVal = ExceptionHelper.handleHttpException(ex);
+        }
+
+        return retVal;
+    }
+
+    @Override
+    public RepoGenericResponse<StatefulNodeGetStatusConfig> getNodeStatus(String nodeId, String authToken, String account) {
+
+        RepoGenericResponse<StatefulNodeGetStatusConfig> retVal;
+
+        try {
+            ApiStatefulNodeGetStatusConfig apiGetNodeStatus  = SpotinstAzureStatefulNodeService
+                    .getNodeStatus(nodeId, authToken, account);
+
+            StatefulNodeGetStatusConfig getStatefulNodeStatus = StatefulNodeConverter.toBl(apiGetNodeStatus);
+            retVal = new RepoGenericResponse<>(getStatefulNodeStatus);
+        }
+        catch (SpotinstHttpException ex) {
+            retVal = ExceptionHelper.handleHttpException(ex);
+        }
+
+        return retVal;
+    }
+
+
+    @Override
+    public RepoGenericResponse<StatefulNode> updateNode(StatefulNode NodeToUpdate ,String nodeId, String authToken, String account) {
```

Comment:
Why input parameter and response type are same?

Reply:
Update Request and Response attributes same like we have for Create API

Reply:
Both are same, so will reuse that same class for response also.